src/Files.App/Dialogs/CloneRepoDialog.xaml

```diff
@@ -0,0 +1,30 @@
+﻿<!--  Copyright (c) Files Community. Licensed under the MIT License.  -->
+<ContentDialog
+	x:Class="Files.App.Dialogs.CloneRepoDialog"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:converters="using:CommunityToolkit.WinUI.Converters"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:helpers="using:Files.App.Helpers"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	Title="Clone repo"
+	HighContrastAdjustment="None"
+	IsPrimaryButtonEnabled="{x:Bind ViewModel.CanCloneRepo, Mode=OneWay}"
+	PrimaryButtonCommand="{x:Bind ViewModel.CloneRepoCommand, Mode=OneWay}"
+	PrimaryButtonStyle="{ThemeResource AccentButtonStyle}"
+	PrimaryButtonText="Clone"
+	RequestedTheme="{x:Bind RootAppElement.RequestedTheme, Mode=OneWay}"
+	SecondaryButtonText="Cancel"
+	Style="{StaticResource DefaultContentDialogStyle}"
+	mc:Ignorable="d">
+
+	<Grid Width="340">
+		<StackPanel Spacing="8">
+			<TextBox
+				Header="Repository URL"
+				PlaceholderText="https://github.com/files-community/Files"
+				Text="{x:Bind ViewModel.RepoUrl, Mode=TwoWay}" />
```

Comment:
```suggestion
				Text="{x:Bind ViewModel.RepoUrl, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
```

Consider adding this to update the property when the user types anything. Currently, iţ only updates when you click away (either on `Clone` or `Cancel`)

Reply:
Thanks! I was looking for a way to improve the validation.